src/sentry/static/sentry/app/actionCreators/events.tsx

```diff
@@ -1,24 +1,40 @@
 import {canIncludePreviousPeriod} from 'app/views/events/utils/canIncludePreviousPeriod';
 import {getPeriod} from 'app/utils/getPeriod';
+import {EventsStats, Organization} from 'app/types';
 
-const BASE_URL = org => `/organizations/${org.slug}/events-stats/`;
+const BASE_URL = (org: Organization) => `/organizations/${org.slug}/events-stats/`;
+
+type Options = {
+  organization: Organization;
+  project?: number[];
```

Comment:
I think we don't need to mark this (and some of the items below) as optional since we always pass them in here and (i'm guessing) they are probably required downstream

Reply:
You can have an empty list/bull projects when all projects are implicitly selected.

Reply:
Some of them are required, but the component has defaults if they are, doesn't that make them optional?